components/dot_menu/index.js

```diff
@@ -16,7 +17,10 @@ import {
 import DotMenu from './dot_menu.jsx';
 
 function mapStateToProps(state) {
+    const teamId = getCurrentTeamId(state);
+
     return {
+        teamId,
```

Comment:
Minor nit: Since `teamId` is just used for the return value, you may have it directly. 
```suggestion
        teamId: getCurrentTeamId(state),
```